cli/installer/install-azd.ps1

```diff
@@ -315,106 +328,43 @@ try {
         reportTelemetryIfEnabled 'InstallFailed' 'DownloadFailed' @{ downloadUrl = $downloadUrl }
         exit 1
     }
-
-    Write-Verbose "Decompressing artifacts" -Verbose:$Verbose
-    if ($extension -eq 'zip') {
-        try {
-            Expand-Archive -Path $releaseArtifactFilename -DestinationPath $tempFolder/decompress
-        } catch {
-            Write-Error "Cannot expand $releaseArtifactFilename"
-            Write-Error $_
-            reportTelemetryIfEnabled 'InstallFailed' 'ArchiveDecompressionFailed'
-            exit 1
-        }
-    } elseif ($extension -eq 'tar.gz') {
-        Write-Verbose "Extracting to $tempFolder/decompress"
-        New-Item -ItemType Directory -Path "$tempFolder/decompress" | Out-Null
-        Write-Host "tar -zxvf $releaseArtifactFilename -C `"$tempFolder/decompress`" $binFilename"
-        tar -zxvf $releaseArtifactFilename -C "$tempFolder/decompress" $binFilename
-
-        if ($LASTEXITCODE) {
-            Write-Error "Cannot expand $releaseArtifactFilename"
-            reportTelemetryIfEnabled 'InstallFailed' 'ArchiveDecompressionFailed'
-            exit $LASTEXITCODE
-        }
-    } else { 
-        Write-Verbose "Decompression not required" -Verbose:$Verbose
-    }
-
-    
+   
 
     try {
-        if (isLinuxOrMac) {
-            if ($IsMacOS -and !$SkipVerify) {
-                Write-Verbose "Verifying signature of $binFilename" -Verbose:$Verbose
-                $codeSignOutput = codesign -v "$tempFolder/decompress/$binFilename" 2>&1
-                if ($LASTEXITCODE) {
-                    Write-Error "Could not verify signature of $binFilename, error output:"
-                    $codeSignOutput |  ForEach-Object {
-                        if ($_ -is [System.Management.Automation.ErrorRecord]) {
-                          Write-Error $_
-                        } else {
-                          Write-Host $_
-                        }
-                    }
+        if (!$SkipVerify) {
+            try {
+                Write-Verbose "Verifying signature of $releaseArtifactFilename" -Verbose:$Verbose
+                $signature = Get-AuthenticodeSignature $releaseArtifactFilename
+                if ($signature.Status -ne 'Valid') {
+                    Write-Error "Signature of $releaseArtifactFilename is not valid"
                     reportTelemetryIfEnabled 'InstallFailed' 'SignatureVerificationFailed'
                     exit 1
                 }
+            } catch {
+                Write-Error "Could not verify signature of $releaseArtifactFilename"
+                Write-Error $_
```

Comment:
I'd collapse these to avoid complicating the `$Error` stream e.g.,
```pwsh
Write-Error "Could not verify signature of $releaseArtifactFilename: $($_.Message)" -Exception $_
```